### DISCUSSION TITLE
ElasticSearch integration

### DISCUSSION THREAD

**Discussion by jsmit99** [beyondcode/herd-community#1182] [Ideas]
It would be great if we can add Elasticsearch v7 / v8 running in the services.

**Comment by JoostWan** [comment]
We prefer Opensearch

**Reply by jsmit99** [reply]
I agree, opensearch would be great. Though that one also is not available in Herd unfortunately.

**Comment by rav-redchamps** [comment]
It will be great to have Opensearch under Herd services that way we can say Herd will support Magento2 fully.
